### PR TITLE
Adding Eventing Example in eventing section

### PR DESCRIPTION
**Pull request by salaboy**
Fixes #issue-number or description of the problem the PR solves

## Proposed Changes 

-
-
-

### CODE REVIEW

File: community/samples/README.md [hunk 1]
```diff
@@ -20,8 +20,7 @@ Knative Serving sample apps.
 
 #### Eventing and Eventing Resources samples
 
-- _Be the first to contribute an Eventing or Eventing Sources code sample to the
-  community collection._
+| Hello World Eventing | A quick introduction to Knative Eventing with HTTP and Kafka Source binding using Cloud Events | [Blog Post](https://salaboy.com/2020/02/20/getting-started-with-knative-2020/), [Event Producer using Spring Cloud Streams & Kafka](https://github.com/salaboy/knative-spring-cloud-event-producer) and [HTTP Cloud Event producer using Spring Boot](https://github.com/salaboy/knative-event-producer) |
```
Comment (matzew) [inline community/samples/README.md:23]: the formatting is a bit odd ?

Comment (salaboy) [reply community/samples/README.md:23]: @matzew I've copied the table from the previous section. It has the same formatting. Can you suggest me a better formatting for that?

Comment (sebgoa) [reply community/samples/README.md:23]: it is not just odd, it does not render properly as far as I can see: https://github.com/knative/docs/blob/fb6bbe4c00c9f76f63b4438d51a686beb4205b90/community/samples/README.md

Comment (sebgoa) [reply community/samples/README.md:23]: first the section heading should only be 3 # , then you are missing top level categories underlined b y dashes.

Comment (salaboy) [reply community/samples/README.md:23]: @sebgoa thanks for letting me know... let me fix that issue.

File: community/samples/README.md [hunk 2]
```diff
@@ -18,10 +18,11 @@ Knative Serving sample apps.
 | Hello World | A quick introduction to Knative Serving that highlights how to deploy an app. | [Clojure](./serving/helloworld-clojure/README.md), [Dart](./serving/helloworld-dart/README.md), [Elixir](./serving/helloworld-elixir/README.md), [Haskell](./serving/helloworld-haskell/README.md), [Java - Micronaut](./serving/helloworld-java-micronaut/README.md), [Java - Quarkus](./serving/helloworld-java-quarkus/README.md), [R - Go Server](./serving/helloworld-r/README.md), [Rust](./serving/helloworld-rust/README.md), [Swift](./serving/helloworld-swift/README.md), [Vertx](./serving/helloworld-vertx/README.md) |
 | Machine Learning | A quick introduction to using Knative Serving to serve machine learning models | [Python - BentoML](./serving/machinelearning-python-bentoml)
 
-#### Eventing and Eventing Resources samples
+### Eventing and Eventing Resources samples
 
-- _Be the first to contribute an Eventing or Eventing Sources code sample to the
-  community collection._
+Knative Eventing samples.
```
Comment (mattmoor) [inline community/samples/README.md:23]: Remove trailing whitespace:
```suggestion
Knative Eventing samples.
```

File: community/samples/README.md [hunk 3]
```diff
@@ -18,10 +18,12 @@ Knative Serving sample apps.
...
```
Comment (mattmoor) [inline community/samples/README.md:23]: Remove trailing whitespace:
```suggestion
Knative Eventing samples.
```

**Comment by salaboy** [comment]
@matzew and @sebgoa not it is looking better :) thanks for reviewing

**Comment by evankanderson** [comment]
I think this page is intended to organize [the samples under `community/samples`](https://github.com/knative/docs/tree/master/community/samples) in this repo.

I don't know if you'd rather rework your sample to check it in, or if you want to set up a section for "samples hosted elsewhere" in this doc.

**Comment by jonatasbaldin** [comment]
I believe would be better to work the sample into the `community/samples`, making it easier for people to contribute or point any issues 🤔

**Comment by salaboy** [comment]
@evankanderson @jonatasbaldin I totally agree.. now that I understand what this page is about. I will close this PR and work on submitting an example that can live inside community/samples.

**Pull request closed without merging**